pkg/exchange/okex/convert.go

```diff
@@ -286,9 +286,10 @@ func toGlobalOrder(okexOrder *okexapi.OrderDetails) (*types.Order, error) {
 }
 
 func toGlobalTrade(orderDetail *okexapi.OrderDetails) (*types.Trade, error) {
-	tradeID, err := strconv.ParseInt(orderDetail.LastTradeID, 10, 64)
+	// Should use tradeId, but okex use billId to perform pagination, so use billID as tradeID instead.
+	billID, err := strconv.ParseInt(orderDetail.BillID, 10, 64)
```

Comment:
Can the bill id be found at okx console?

Reply:
console? you mean web page? No, You can't find billId in official site, actually you can't find orderId and tradeId either.